apps/console/src/features/applications/pages/applications.tsx

```diff
@@ -316,14 +333,17 @@ const ApplicationsPage: FunctionComponent<ApplicationsPageInterface> = (
                     onItemsPerPageDropdownChange={ handleItemsPerPageDropdownChange }
                     onPageChange={ handlePaginationChange }
                     onSortStrategyChange={ handleListSortingStrategyOnChange }
-                    showPagination={ (!isApplicationListRequestLoading && appList?.totalResults !== 0) }
+                    showPagination={ true }
```

Comment:
Shouldn't we revert this back to the previous state?

Reply:
That condition is not required as the page loading is handled in the high level component. And this condition causes the pagination buttons to disable at some times.